src/main/core/worker.c

```diff
@@ -625,34 +630,41 @@ static void _worker_shutdownHost(Host* host, void* _unused) {
 }
 
 guint32 worker_getNodeBandwidthUp(in_addr_t ip) {
```

Comment:
optional, since it's orthogonal to this change, but it'd be nice if these functions included the units. e.g. worker_getNodeBandwidthUpKiBps

Reply:
Added another commit with this change.